countdown-timer.js

```diff
@@ -0,0 +1,131 @@
+const startButton = document.querySelector("#start-button");
+const pauseButton = document.querySelector("#pause-button");
+const resetButton = document.querySelector("#reset-button");
+const countdownTimer = document.querySelector("#countdown-timer");
+
+const hourInput = document.querySelector("#countdown-hour-input");
+const minuteInput = document.querySelector("#countdown-minute-input");
+const secondInput = document.querySelector("#countdown-second-input");
+
+hourInput.addEventListener("input", (event) => {
+  let value = Number(event.target.value);
+  if (!Number.isInteger(value)) {
+    alert("Please enter an integer.");
+    event.target.value = "";
+  }
+  if (value < 0 || value > 99) {
+    alert("Please enter an Integer between 0 to 99.");
+    event.target.value = "";
+  }
+});
+
+function minuteSecondInputLimit(event) {
+  let value = Number(event.target.value);
+  if (!Number.isInteger(value)) {
+    event.target.value = Math.floor(value);
+  }
+  if (value < 0 || value > 59) {
+    alert("Please enter an Integer between 0 to 59.");
+    event.target.value = "";
+  }
+}
+
+minuteInput.addEventListener("input", minuteSecondInputLimit);
+secondInput.addEventListener("input", minuteSecondInputLimit);
+
+function formatTime(seconds) {
+  const hour = String(parseInt(seconds / 3600)).padStart(2, "0") || 0;
```

Comment:
```suggestion
  const hour = String(parseInt(seconds / 3600)).padStart(2, "0");
```
I don't think you need these default values.